operate/schema/src/main/java/io/camunda/operate/schema/elasticsearch/ElasticsearchSchemaManager.java

```diff
@@ -280,8 +282,10 @@ public IndexMapping getExpectedIndexFields(final IndexDescriptor indexDescriptor
   }
 
   private String settingsTemplateName() {
-    final OperateElasticsearchProperties elsConfig = operateProperties.getElasticsearch();
-    return String.format("%s_template", elsConfig.getIndexPrefix());
+    final String indexPrefix = getIndexPrefix();
```

Comment:
I guess you checked that it is named like this?

Reply:
not sure what you mean, it was named "operate_template" previously, now with the default empty prefix it will be "operate_template" still.